eng/tools/mcp/azure-sdk-java-mcp/src/update-changelog-md.spec.ts

```diff
@@ -0,0 +1,141 @@
+import { describe, it, expect } from "vitest";
+import { updateChangelogText } from "./update-changelog-md.js";
+
+describe("updateChangelogText", () => {
+    it("should be defined", () => {
+        expect(updateChangelogText).toBeDefined();
+        expect(typeof updateChangelogText).toBe("function");
+    });
+
+    it("should insert new changelog content after the first version subsection", () => {
+        const oldChangelog = `# Changelog
+
+## 1.2.0 (2025-01-01)
```

Comment:
Not sure how this text will be used but will the tool adapt the response to use the right version number, date etc?